improver_tests/nowcasting/utilities/test_FillRadarHoles.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Module name "test_FillRadarHoles" doesn't conform to '(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5409292)